### PR TITLE
community/wrk: upgrade to 4.0.2

### PR DESCRIPTION
**Pull request by itoffshore**
https://github.com/wg/wrk/blob/master/CHANGES

### CODE REVIEW

File: community/wrk/APKBUILD [hunk 1]
```diff
@@ -2,27 +2,30 @@
...
 pkgdesc="wrk is a modern HTTP benchmarking tool"
 url="https://github.com/wg/wrk"
 # requires -march=i586 to build on x86
 arch="x86_64"
 license="ASL 2.0"
 depends=""
 depends_dev=""
-makedepends="libressl-dev"
-install=""
+options="!check"
```
Comment (jirutka) [inline community/wrk/APKBUILD]: **Why** it's disabled?

Comment (itoffshore) [reply community/wrk/APKBUILD]: I tried all of the following:
```
make test
make tests
make check
make checks
```
& ran `grep` in `src` looking for `test` & `check` but didn't find anything

Comment (jirutka) [reply community/wrk/APKBUILD]: Then add a comment like "upstream doesn't provide tests" after the option=.

Comment (itoffshore) [reply community/wrk/APKBUILD]: comment added in https://github.com/alpinelinux/aports/pull/2730

**Pull request closed without merging**